libs/designsystem/dropdown/src/dropdown.component.ts

```diff
@@ -525,6 +516,124 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
     }
   }
 
+  private preventDefaultAndStopImmediatePropagation(event: KeyboardEvent) {
+    event.stopImmediatePropagation();
+    event.preventDefault();
+  }
+
+  private searchBuffer: string = '';
+  private searchBufferTimeout: ReturnType<typeof setTimeout>;
+  private SEARCH_BUFFER_DELAY = 500; // ms
```

Comment:
Consider moving these all the way to the top of the class together with other field variables.